src/macros.js

```diff
@@ -659,6 +659,12 @@ defineMacro("\\mod", "\\allowbreak" +
     "\\mathchoice{\\mkern18mu}{\\mkern12mu}{\\mkern12mu}{\\mkern12mu}" +
     "{\\rm mod}\\,\\,#1");
 
+// \pmb    --   A simulation of bold.
+// It works by typesetting three copies of the argument with small offsets.
+// Ref: a rather lengthy macro in ambsy.sty
+defineMacro("\\pmb", "\\html@mathml{\\@binrel{#1}{\\mathrlap{\\mathrlap{#1}" +
+    "\\mkern0.4mu\\raisebox{0.4mu}{$#1$}}{\\mkern0.8mu#1}}}{\\mathbf{#1}}");
```

Comment:
This is much easier to read now, thanks!

I have one remaining question: why is the `\mathrlap{#1}` nested inside another `\mathrlap`?  I imagine it could also go at the very beginning, like `\mathrlap{#1}\mathrlap{\mkern0.4mu...}`.  Any particular reason to prefer one over the other?